src/conspiracies/docprocessing/coref/coref_model.py

```diff
@@ -37,16 +41,26 @@ class CoreferenceModel(Predictor):
     def __init__(
         self,
         model_path: Union[Path, str, None] = None,
+        language: str = "da",
         device: int = -1,
         open_unverified_connection: bool = False,
         **kwargs,
     ) -> None:
         if model_path is None:
-            model_path = download_model(
-                "da_coref_twitter_v1",
-                open_unverified_connection=open_unverified_connection,
-            )
+            if language == "da":
+                model_path = download_model(
+                    "da_coref_twitter_v1",
+                    open_unverified_connection=open_unverified_connection,
+                )
+            elif language == "en":
+                model_path = (
+                    "https://storage.googleapis.com/allennlp-public-models/"
+                    "coref-spanbert-large-2021.03.10.tar.gz"
+                )
```

Comment:
Ideally, we just want to allow a model path I assume (though suspecting that we will switch to spacy's coreference model in the future I wouldn't worry too much about it)

Reply:
Not a bad idea! I'll take a look at it.